python/mlc_llm/quantization/group_quantization.py

```diff
@@ -429,109 +287,6 @@ def _quantize(  # pylint: disable=too-many-locals
             scale = topi.transpose(scale)
         return quantized_weight, scale
 
-    def _quantize_float8(  # pylint: disable=too-many-locals
```

Comment:
FYI @csullivan @JosephTheOctonaut , it looks like this merge removed some of the FP8 quantization functions, which may need to be restored.

Reply:
Our FP8 quantization flow is mainly using functions defined in mlc-serve/slm, which unfortunately is now diverged from the upstream copy. E.g., for the specific function `_quantize_float8`, it lives in slm/quantization/per_tensor_quantization.py for our flow. I'm not totally clear on the upstream-only flow for FP8, but @vinx13 has been updating that over time, so I'm assuming it's intact.

We probably want to visit unifying the flows down the line, if reasonable, to avoid the duplication and conflicts that Sung pointed out.